docs/maya/tools.md

```diff
@@ -77,7 +77,13 @@ In the specific case of muscle deformers, the too will display the following att
     - Masses are set to 1.0 by default. This will mean that by default the solver will consider that the muscle has a uniform mass.
 <!-- Note for later to update this information once the Global Damping attribute is added-->
 <!-- Note for later to update this information once the Slide On Geometry Damping attribute is added-->
-  - **Slide on Segment Constraints**
+  - **Slide On Geomety**
+    - If this attribute is selected, a list widget is shown with the names of the targets connected to the deformer (Figure 8).
```

Comment:
a) Feel free to remove the above to this line documentation notes.
b) 
```suggestion
  - **Slide On Geometry**
    - If this attribute is selected, a list widget is shown with the names of the targets connected to the deformer (Figure 8).
```

Reply:
Done.